packages/fast-development-site-react/app/app.tsx

```diff
@@ -31,7 +31,7 @@ export interface AppState {
 const themes: Theme[] = [
     { id: "Foo", displayName: "Foo", background: "#000" },
     { id: "Bar", displayName: "Bar", background: "#FFF" },
-    { id: "Rumple", displayName: "Rumple", background: "#333" },
+    { id: "Rumple", displayName: "Rumple", background: "#222" },
```

Comment:
Was this an intentional change?

Reply:
Yes, it allows the text to be visible on all test themes.

Reply:
But, it's hard coded?  Should it be using our color library so it's dynamic?

Reply:
Not for the dev site app - it's just a test app showing how to set up a dev site. I'm sure @janechu set it up this way intentionally.

Reply:
I didn't set this particular part of it up, but yes its just a test app for the development site, it can't take a dependency, dev or otherwise, on any other library in our repository because it would be circular. Having said that I would prefer if we are adding to the array that it follows the current naming convention so the next item should be "Bat"